docs/io/output/index.rst

```diff
@@ -12,5 +12,6 @@ In addition to the widgets, TARDIS can output information in several other forms
     to_hdf
     callback
     vpacket_logging
-    progress_bars
     rpacket_tracking
+    progress_bars
```

Comment:
okay this shouldn't be part of the PR, right?

Reply:
This was rebased. I will check this.